linux_os/guide/system/auditing/auditd_configure_rules/directory_access_var_log_audit/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Search /etc/audit/rules.d for audit rule entries
+  find:
+    paths: /etc/audit/rules.d
+    recurse: false
+    contains: ^.*dir=/var/log/audit/.*$
+    patterns: '*.rules'
+  register: find_var_log_audit
+
+- name: Use /etc/audit/rules.d/access-audit-trail.rules as the recipient for the rule
+  set_fact:
+    all_files:
+    - /etc/audit/rules.d/access-audit-trail.rules
+  when: find_var_log_audit.matched == 0
+
+- name: Use matched file as the recipient for the rule
+  set_fact:
+    all_files:
+    - '{{ find_var_log_audit.files | map(attribute=''path'') | list | first }}'
+  when: find_var_log_audit.matched &gt; 0
```

Comment:
There is no need to escape `>` here.
```
TASK [Use matched file as the recipient for the rule] *************************************************************************************************************************************************************
fatal: [192.168.122.84]: FAILED! => {"msg": "The conditional check 'find_var_log_audit.matched &gt; 0' failed. The error was: template error while templating string: unexpected char '&' at 33. String: {% if find_var_log_audit.matched &gt; 0 %} True {% else %} False {% endif %}\n\nThe error appears to have been in '/home/wsato/git/scap-security-guide/tests/logs/rule-custom-2019-03-06-1019/xccdf_org.ssgproject.content_rule_directory_access_var_log_audit.yml': line 51, column 7, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n      - low_disruption\n    - name: Use matched file as the recipient for the rule\n      ^ here\n"}
	to retry, use: --limit @/home/wsato/git/scap-security-guide/tests/logs/rule-custom-2019-03-06-1019/xccdf_org.ssgproject.content_rule_directory_access_var_log_audit.retry
```

Reply:
@yuumasato huh.... no idea how that got there. Fixed.